src/Sulu/Bundle/AdminBundle/Admin/View/PreviewFormViewBuilder.php

```diff
@@ -38,6 +38,13 @@ public function setResourceKey(string $resourceKey): PreviewFormViewBuilderInter
         return $this;
     }
 
+    public function setPreviewResourceKey(string $previewResourceKey): PreviewFormViewBuilderInterface
```

Comment:
This would allow to start the preview also for the permission tab inside of the contact (but disable the update functionality)

Reply:
only as an example

Reply:
or better example is permissions in pages (has pushed that in this branch)

Reply:
good idea!